easybuild/easyconfigs/r/RELION/sbatch_slurm_gpu.bash

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+#SBATCH -J XXXnameXXX
+# Unset XXX...XXX values will be empty, and SBATCH without argument is ignored.
+#SBATCH -t XXXextra1XXX:00:00
+#SBATCH --gres=gpu:XXXextra4XXX
+#SBATCH -n XXXmpinodesXXX
+#SBATCH -c XXXthreadsXXX
+#SBATCH -e XXXerrfileXXX
+#SBATCH -o XXXoutfileXXX
+#SBATCH -A XXXextra2XXX
```

Comment:
If XXXextra2XXX is empty this will generate a strange error to the user, and so will the #SBATCH -t XXXextra1XXX:00:00 if extra1 is unset.
And by strange I meant really strange ones. Slurm isn't the best at producing good error messages at all times.

That's why I made those two explicitly include -A/-t in the variable, i.e. XXXaccountXXX and XXXtimelimitXXX
Then the user will at least get a good and understandable error.

Also, for us at least, the --gres part either has to be complete or not there at all. So I made it XXXgpuspecXXX. That way we can use a single cpu/gpu common template file.
It's something that may or may not be a complicated thing to specify and is highly site dependent.

Reply:
I updated those non-positional extra parameters so that they are not empty by default. This will avoid triggering unintended errors. However, I expect sysadmins installing this software to adapt the settings of easyconfig to their needs.

I agree that your patch does a much better job at providing custom parameters for Slurm, but I really think that it is not feasible to achieve an easyconfig of RELION that works out of the box for all sites. For instance, your patch won't work with Torque and I think that it is not guaranteed that it will work on all systems using Slurm. I also had to adapt the template for Torque to make it work in our systems, but in my opinion that should be expected, as simple things as queue names or accounting do vary. That's why my goal is to have something that provides a good starting point for EB users and that can be easily adapted.